src/org/usfirst/frc/team2335/robot/commands/Turn.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class Turn extends Command {
+
+    public Turn() {
+        requires(Robot.driveTrain);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	Robot.driveTrain.turn(Robot.oi.getAxis(Robot.ROTATE, 0.75));
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return false;
+    }
+
+    // Called once after isFinished returns true
+    protected void end() {
+    	Robot.driveTrain.stopDrive();
+    }
+
+    // Called when another command which requires one or more of the same
+    // subsystems is scheduled to run
+    protected void interrupted() {
+    }
+}
```

Comment:
Same thing for this file as the Strafe command, any comments you want to keep. If not the spacing and comments need to be fixed.